.github/workflows/pkgdown.yaml

```diff
@@ -73,12 +73,17 @@ jobs:
           pkgdown::build_site()
         shell: Rscript {0}
 
+      - name: Create index file
+        if: contains(env.isPush, 'false')
+        run: |
+          cp fake-index.html ./docs/index.html
+
       - name: Deploy to Netlify
         if: contains(env.isPush, 'false')
         id: netlify-deploy
         uses: nwtgck/actions-netlify@v1.1
         with:
-          publish-dir: './docs/dev'
+          publish-dir: './docs'
```

Comment:
Can you explain more why this has changed?

Reply:
We need one search index per site, so one for the production website and one for development website.
I want the PR preview to mimick what happens in real life (for pkgdown.r-lib.org, and any package) which means that the search index for the development website is at `/dev/search.json`. If we were still deploying `./docs/dev.json` we'd expect the search index at `/dev/search.json` but it would be at `/search.json`.
I think this new preview setup will also help us make better decisions for e.g. the 404 page (that contrary to the search index cannot be specific to one of the development/production websites, as far as I understand).

Reply:
Ok, makes sense.